FaunaDB.Client/Client/DefaultClientIO.cs

```diff
@@ -27,25 +27,31 @@ class DefaultClientIO : IClientIO
         readonly AuthenticationHeaderValue authHeader;
 
         private LastSeen lastSeen;
+        private Version httpVersion;
 
         public const string StreamingPath = "stream";
         public const HttpMethodKind StreamingHttpMethod = HttpMethodKind.Post;
 
-        internal DefaultClientIO(HttpClient client, AuthenticationHeaderValue authHeader, LastSeen lastSeen, Uri endpoint, TimeSpan? timeout)
+        internal DefaultClientIO(HttpClient client, AuthenticationHeaderValue authHeader, LastSeen lastSeen, Uri endpoint, TimeSpan? timeout, Version httpVersion)
         {
             this.client = client;
             this.authHeader = authHeader;
             this.lastSeen = lastSeen;
             this.endpoint = endpoint;
             this.clientTimeout = timeout;
+#if NETSTANDARD2_1
+            this.httpVersion = httpVersion == null ? new Version(2, 0) : httpVersion;
+#else
+            this.httpVersion = httpVersion == null ? new Version(1, 1) : httpVersion;
+#endif
```

Comment:
oh, nice catch!

Reply:
btw... this is going to be valid for standards >= 2.1, right? 
(that framework/standard matrix is always confusing me)

Reply:
yep, they added http/2 support only starting from .net core 3.0, which is standard 2.1,
the above standards will also support http/2 (there is not going to be standards notion anymore after 2.1 though :) they moved to .net 5, .net 6 and so on, but that can be considered as "above")